### ISSUE TITLE
Support sorting by and editing custom tags

### ISSUE THREAD

**Issue by ellietheoracle** [mirukana/mirage#71]
Essentially, being able to sort rooms by manually created tags would be fantastic .

https://matrix.org/docs/spec/client_server/latest#m-tag
https://github.com/turt2live/matrix-tag-manager
https://matrix-nio.readthedocs.io/en/latest/nio.html?highlight=tags#nio.events.account_data.TagEvent

**Comment by Cadair** [comment]
Being able to hide rooms from the list based on tag would also be useful. i.e. personally most of the time I don't want my "Low Priority" tagged rooms at the top of my room list.

**Comment by ellietheoracle** [comment]
right, if they were under a drop down, like rooms under an account are, that would be ideal .

**Comment by ellietheoracle** [comment]
so basically, 
![render](https://user-images.githubusercontent.com/20211219/85925551-aef5c380-b85e-11ea-838c-69ef8ff5b21e.png)
like this

**Comment by ellietheoracle** [comment]
obviously it's a pretty crappy mockup, but it gets the point across .

**Comment by ellietheoracle** [comment]
![Screenshot from 2020-06-27 10-14-03](https://user-images.githubusercontent.com/20211219/85925592-f2e8c880-b85e-11ea-8ab1-23db44ce887f.png)
this could have another tab for tags, to rearrage, add, delete tags, etc .

**Comment by mirukana** [comment]
Duplicate of #6

**Comment by ellietheoracle** [comment]
is it actually a duplicate ? this is specifically about custom tags, not just the standard "low priority", "dm", etc .

**Comment by ellietheoracle** [comment]
![Screenshot from 2020-06-27 11-18-08](https://user-images.githubusercontent.com/20211219/85926859-e9b02980-b867-11ea-8b48-e7c9e9070954.png)
e.g. in riot

**Comment by mirukana** [comment]
> is it actually a duplicate ? this is specifically about custom tags, not just the standard "low priority", "dm", etc .

Low priority and other standard tags work just the same way with the same APIs, the only difference from custom tags is they have a `m.` prefix.

**Comment by ellietheoracle** [comment]
yeah, but allowing the user to manage them requires a completely separate UI .

**Comment by ellietheoracle** [comment]
for the standard 3 tag approach, a simple right click menu is enough to sort them properly, but that's absolutely not scalable .

**Comment by j0w0** [comment]
would find this useful for all the discord servers i have to use :p

**Comment by theotheroracle** [comment]
should note that custom tags have been more or less officially deprecated in favour of spaces
